app/javascript/spec/workflow-repository-form/helpers.spec.js

```diff
@@ -0,0 +1,21 @@
+import { customUrlValidator } from '../../components/ansible-repository-form/helpers';
```

Comment:
This should use a shared helper (or at least the workflow-repository-form/helpers until a shared helper is created).